### PR TITLE
Make package

### PR DESCRIPTION
**Pull request by corinnebosley**
Make package importable by restructuring (a bit) and doing some other bits.

### CODE REVIEW

**Comment by corinnebosley** [comment]
Might be best to review and merge license before merging this, just so that we can get everything in the right place, but I have tested the build using pip in a clean environment and it all seems to work.

File: setup.py
Comment (crbunney) [inline setup.py:6]: I don't believe this is required with recent versions of `setuptools` (although I don't know how recent that change is, nor do we have a formal stance on what versions we're committing to supporting)

**Review by crbunney — APPROVED** [review]
Looks comprehensive and good. I merged #9  if you wanted to retest with that in place

**Comment by corinnebosley** [comment]
Thanks for merging #9 @crbunney , I will retest with that in place and hopefully get back to you shortly.

**Comment by corinnebosley** [comment]
@crbunney I have added license and re-tested, all seems fine.  No license is in this PR, but I suppose that's just because you've already merged it.  I think this should all be good now.

@lucascatherine It would be good to get your go-ahead on this before I merge.  I have tested in a clean environment, all builds fine.

**Comment by corinnebosley** [comment]
So it turns out that although the repo could be successfully packaged, it wasn't actually importable until these changes.
Import using 'import cap_sample_data'.

I may need to make some file structure changes in a further PR (for example, I am uneasy about shapefiles being on the top level, this may mean they are not importable or useable, and also it might be useful to add a symlink on the top level so that we can import using 'import cap_sample_data.sample_data', but right now I don't know how to do that.

I could do with this being merged (and imported into your respective cap_envs, should you use them) so that I can write tests that will pass.  That being the case (and with me not being around until next Monday), please merge ASAP.

**Comment by corinnebosley** [comment]
As a further note to this PR, once this is merged I may have to change settings in ALL tests in cap, or maybe just this file (https://github.com/ADAQ-AQI/clean-air-project/blob/master/conftest.py) for the tests to pass again.

**Comment by corinnebosley** [comment]
This is it guys, this one works and all the tests pass!!! I have moved all data files into cap_sample_data/sample_data and changed all the bits in the pyproject.toml and stuff to work properly, so now you can import the module using "import cap_sample_data" and get the data paths using "cap_sample_data.path" (this is what has changed in the conftest.py in my next PR to cap).

I am happy with this; ready to merge upon approval from @lucascatherine .

**Comment by lucascatherine** [comment]
Looks good to me.

**Review by lucascatherine — APPROVED** [review]
All looks good.

**Pull request merged**
